website/components/i18n/en.json

```diff
@@ -6,12 +6,10 @@
   "babel-loader-remark": "Be cautious of the performance impact of babel-loader",
   "html-webpack-plugin-desc": "Support for this plugin was implemented in v0.3.3, please upgrade the Rspack version to use it.",
   "sentry_webpack-plugin-desc": "Support for the plugin version v1.20.1 and above has been implemented in v0.3.3",
-  "define-plugin-desc": "Support for this plugin was implemented in v0.3.3, before v0.3.3 you could use builtins.define instead.",
-  "copy-plugin-desc": "Use builtins.copy or rspack.CopyRspackPlugin instead.",
-  "banner-plugin-desc": "Support for this plugin was implemented in v0.3.3, before v0.3.3 you could use builtins.banner instead.",
-  "ignore-plugin-desc": "Recommend using rspack.IgnorePlugin instead.",
-  "mini-css-extract-plugin-desc": "use experiments.css or rspack.CssExtractRspackPlugin instead",
-  "terser-webpack-plugin-desc": "Use rspack.SwcJsMinimizerRspackPlugin instead.",
-  "progress-plugin-desc": "use builtins.progress instead",
-  "tsconfig-paths-webpack-plugin-desc": "use resolve.tsconfigPath instead"
+  "copy-plugin-desc": "Use `builtins.copy` or `rspack.CopyRspackPlugin` instead.",
```

Comment:
It would be helpful if we can link to `rspack.CopyRspackPlugin` or `builtins.copy` here.